airflow/assets/__init__.py

```diff
@@ -272,10 +277,10 @@ def _set_extra_default(extra: dict | None) -> dict:
 class Asset(os.PathLike, BaseAsset):
     """A representation of data dependencies between workflows."""
 
-    name: str = attr.field()
-    uri: str = attr.field()
-    group: str = attr.field()
-    extra: dict[str, Any] = attr.field()
+    name: str
```

Comment:
I guess it's due to the `__init__` thing in the last PR so that we don't need it anymore ?

Reply:
It’s more like because `attr.define` is smart enough to know these are fields automatically. As long as the fields don’t have arguments, the assignment can be omitted.